CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog for Mantarray Waveform Analysis
 =========================================
 
+0.6.11 (2021-03-01)
```

Comment:
if you bump the minor version (the middle one), you reset the patch version (rightmost one) to zero. If you haven't had a chance to take a quick glance through https://semver.org/, it's a worthwhile quick read